### PR TITLE
chore: release

### PR DESCRIPTION
**Pull request by sarahdayan**
This pull request prepares the following release:
| Repository | Branch | Update |
|---|---|---|
| [instantsearch.js](https://github.com/algolia/instantsearch.js) | chore/release-1710928424714 | patch |

### Release Summary
This is going to be published with the following command:

```yarn lerna publish from-package --yes```

### Merging Instructions
When merging this pull request, you need to **Squash and merge** and make sure that the title starts with `chore: release`.
<details>
<summary>See details</summary>

After that, a commit `chore: release` will be added and you or your CI can run `shipjs trigger` to trigger the release based on the commit.
Squash and merge
</details>

---

_This pull request is automatically generated by [Ship.js](https://github.com/algolia/shipjs)_.

### CODE REVIEW

**Review by dhayab — APPROVED** [review]

File: packages/instantsearch.js/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
+
+
+### Bug Fixes
+
+* **vue-instantsearch:** properly add `noRefinement` class on `<ais-refinement-list>` ([#6095](https://github.com/algolia/instantsearch/issues/6095)) ([9ae7c05](https://github.com/algolia/instantsearch/commit/9ae7c05ef6d17bd1d945dc1d31c7d87a262d1e06))
+
+
+
+
+
```
[the remark this answers is not in the document]
Comment (sarahdayan) [reply packages/instantsearch.js/CHANGELOG.md:16]: Yeah I've noticed this, this is likely a weakness in how changelogs are generated.

**Review by Haroenv — COMMENTED** [review]
TBH I'm still not sure this needs a release

**Comment by sarahdayan** [comment]
> TBH I'm still not sure this needs a release

This would release the bugfixes in VIS and RIS. Do you want to wait for other features to land?

**Review by Haroenv — APPROVED** [review]
I'm not blocking the release, it can definitely happen, but as no customers asked for these changes, I didn't see the urgency in releasing out of turn. No big deal

**Comment by sarahdayan** [comment]
I'll cancel it since #6099 fixes an actual customer issue, and let @aymeric-giraudet deploy.

**Pull request closed without merging**